code/modules/antagonists/eldritch_cult/knowledge/mind_lore.dm

```diff
@@ -0,0 +1,115 @@
+	desc = "Upgrade your mansus grasp, granting it the ability to blind a target on hit, as well as increasing the range"
+	cost = 2
+	banned_knowledge = list(/datum/eldritch_knowledge/rust_mark,/datum/eldritch_knowledge/flesh_mark,/datum/eldritch_knowledge/ash_mark)
+	route = PATH_MIND
+	tier = TIER_MARK
+
+/datum/eldritch_knowledge/mind_mark/on_gain(mob/user)
+	. = ..()
+	var/datum/action/cooldown/spell/ranged_touch = locate(/datum/action/cooldown/spell/pointed/mansus_ranged) in user.actions
+	if(ranged_touch)
+		ranged_touch.Remove(user)
+	var/datum/action/cooldown/spell/pointed/mansus_ranged_upgraded/ranged_touch_upgraded = new(user)
+	ranged_touch_upgraded.Grant(user)
+
+/datum/eldritch_knowledge/spell/assault
+	name = "Amygdala Assault"
+	gain_text = "Deep into the dream, you work tirelessly, endlessly. A blighted curse consumes the land, but the scholars of Kos will light the way through the nightmare."
+	desc = "Shoot a single bolt of condensed mental power infront of you, dealing large amounts of brute damage to any targets hit."
+	cost = 1
+	spell_to_add = /datum/action/cooldown/spell/basic_projectile/assault
+	route = PATH_MIND
+	tier = TIER_2
+
+/datum/eldritch_knowledge/spell/famished_roar
+	name = "Famished Roar"
+	gain_text = "Beasts all over the shop. You'll be one of them... Sooner or later... What's that smell? The sweet blood, oh, it sings to me. It's enough to make a man sick..."
+	desc = "An AOE roar spell that sends all near by people and items flying."
+	cost = 1
+	spell_to_add = /datum/action/cooldown/spell/aoe/repulse/frenzied_roar
+	tier = TIER_2
+
+/datum/eldritch_knowledge/mind_blade_upgrade
+	name = "Spine of The Infinite Beast"
+	gain_text = "Curse here, Curse there. Curse for he and she, why care? A bottomless curse, a bottomless sea, source of all greatness, all things that be."
+	desc = "Your mind blade will now inject targets hit with mutetoxin, silencing them."
+	cost = 2
+	banned_knowledge = list(/datum/eldritch_knowledge/rust_blade_upgrade,/datum/eldritch_knowledge/flesh_blade_upgrade,/datum/eldritch_knowledge/ash_blade_upgrade)
+	route = PATH_MIND
+	tier = TIER_BLADE
+
+/datum/eldritch_knowledge/mind_blade_upgrade/on_eldritch_blade(target,user,proximity_flag,click_parameters)
+	. = ..()
+	if(iscarbon(target))
+		var/mob/living/carbon/carbon_target = target
+		carbon_target.reagents.add_reagent(/datum/reagent/toxin/mutetoxin, 2)
+
+/datum/eldritch_knowledge/cerebral_control
+	name = "Full Cerebral Control"
+	gain_text = "Itching on the inside of your skull, like spiders under your skin, runes marked all over your body, etched into your soul. Eyes on the inside to see the truth, to ascend."
+	desc = "Rewire your own brain to partially ignore damage slowdown."
+	cost = 1
+	route = PATH_MIND
+	tier = TIER_3
+
+/datum/eldritch_knowledge/cerebral_control/on_gain(mob/user)
+	. = ..()
+	ADD_TRAIT(user, TRAIT_REDUCED_DAMAGE_SLOWDOWN, type)
+	///add something here
```

Comment:
:3

Reply:
removed the comment (I assume that was the problem)